blocks/extensions/plot.py

```diff
@@ -63,9 +61,7 @@ class Plot(SimpleExtension):
     server_url : str, optional
         Url of the bokeh-server. Ex: when starting the bokeh-server with
         ``bokeh-server --ip 0.0.0.0`` at ``alice``, server_url should be
-        ``http://alice:5006``. When not specified the default configured
-        by ``bokeh_server`` in ``.blocksrc`` will be used. Defaults to
-        ``http://localhost:5006/``.
```

Comment:
Apparently I failed to merge https://github.com/bartvm/blocks/commit/9c8ad24f9d728f1cc534c6fc0d9cd7a6a2991d24 for some reason.